lib/gooddata/bricks/middleware/logger_middleware.rb

```diff
@@ -38,7 +38,8 @@ def call(params)
           FileUtils.mkpath log_directory
           logger = Logger.new("#{log_directory}/#{execution_id}.log")
           logger.level = params['GDC_LOG_LEVEL'] || 'info'
-          logger = MaskLoggerDecorator.new(logger, params)
+          values_to_mask = params['values_to_mask'] || []
+          logger = MaskLoggerDecorator.new(logger, values_to_mask) if values_to_mask.any?
```

Comment:
let's keep the interface of GoodData.logger constant and always initialize MaskLoggerDecorator, even if values_to_mask is an empty array.

Reply:
Done